### PR TITLE
Refactored the main

### PR DESCRIPTION
**Pull request by r0mainK**
Reordered the get_parser function for clarity

### CODE REVIEW

**Comment by r0mainK** [comment]
Should be merged after this is: https://github.com/src-d/ml/pull/215

**Comment by r0mainK** [comment]
EDIT: should not be merged before [this is](https://github.com/src-d/ml/pull/223)

File: apollo/__main__.py [hunk 1]
```diff
@@ -207,13 +211,13 @@ def add_dzhigurda_arg(my_parser):
     add_cassandra_args(evalcc_parser)
 
     # TODO: retable [.....] -> [.] [.] [.] [.] [.]
-
+
```
Comment (vmarkovtsev) [inline apollo/__main__.py]: Dangling whitespace here

**Comment by vmarkovtsev** [comment]
@r0mainK Fix the conflicts and we are ready to go

**Comment by r0mainK** [comment]
@vmarkovtsev i fixed the conflic, but can seem to be able to merge the two commits in 1 with `git rebase HEAD~2`, it shows me other commits --'' also this should not be merged until https://github.com/src-d/ml/pull/223 is meged, since it contains a bunch of changes added there

**Pull request merged**